.github/workflows/ci.yml

```diff
@@ -222,3 +222,55 @@ jobs:
           tool: cargo-workspace-lints
       - name: cargo workspace-lints
         run: cargo workspace-lints
+  geneva-test:
```

Comment:
This test install's bond encoder, and takes around ~9-10 minutes - will make this optional to complete for merging the unrelated pull requests.

Reply:
This is removed with pure bond encoding.